aap-common/proc-aap-pull-command-container-image.adoc

```diff
@@ -4,12 +4,20 @@
 
 Pull the Docker image for the Ansible on Clouds operational container with the same tag version as your deployment.
 
-For example, if your foundation deployment version is 2.2.20230215, you must pull the operational image with tag 2.2.20230221-00.
+For example, if your foundation deployment version is 2.3.20230221-00, pull the `ansible-on-clouds-ops` image with tag 2.3.20230221.
```

Comment:
Should there be a -00 at the end?

Reply:
The tag used to pull will not have -00 (due to downstream build publication limitations in the tooling)